src/components/media/MediaMenu.tsx

```diff
@@ -10,7 +11,15 @@ import { Menu } from '../menu';
 import { bdlGray50 } from '../../styles/variables';
 import './Media.scss';
 
-export interface MediaMenuProps extends PlainButtonProps {
+const messages = defineMessages({
```

Comment:
Can we move this to a `messages.js` file in this directory?

Reply:
do you really want a whole new file for a single variable?
I take this file as reference https://github.com/box/box-ui-elements/blob/master/src/components/time-input/TimeInput.tsx#L15

Reply:
It's less about my personal preference than it is the convention for most (though not all) components in the repository.

Reply:
okas, no problem
just asking because that other file 

going to move it into `messages.js`